packages/components/scripts/jobs/angular/createValidatorDirectives.js

```diff
@@ -0,0 +1,123 @@
+import fs from 'fs/promises';
+import path from 'path';
+import { createHeader, job } from '../shared.js';
+
+const headerComment = createHeader('angular');
+
+/**
+ * Creates the synergy angular module, located at packages/angular/src/modules/synergy.module.ts
+ */
+export const runCreateFValidatorDirectives = job('Angular: Creating ValidatorDirectives...', async (outDir) => {
```

Comment:
There is a small typo in here, `runCreateFValidatorDirectives` should be named `runCreateValidatorDirectives` (without "F")